packages/core/src/common/utils.ts

```diff
@@ -104,3 +104,36 @@ export function throttleEvent(target: EventTarget, eventName: string, newEventNa
     target.addEventListener(eventName, func);
     return func;
 };
+
+/**
+ * Throttle a callback by wrapping it in a `requestAnimationFrame` call. Returns the throttled
+ * function.
+ * @see https://www.html5rocks.com/en/tutorials/speed/animations/
+ */
+export function throttleReactEventCallback(
+    callback: (event: React.SyntheticEvent<any>, ...otherArgs: any[]) => any,
+    preventDefault?: boolean,
```

Comment:
It also makes the lines the _exact_ same length: 🤤

![image](https://user-images.githubusercontent.com/443450/29099292-560aa18a-7c59-11e7-8ae3-1d94e7ac4ee9.png)